src/models/searchservice/response/EmailDirectAnswer.ts

```diff
@@ -0,0 +1,18 @@
+import { FieldValueDirectAnswer } from './FieldValueDirectAnswer';
+import { BuiltInFieldType } from './BuiltInFieldType';
+
+/**
+ * A interface for direct answers {@link FieldValueDirectAnswer} with email values.
+ *
+ * @public
+ */
+export interface EmailDirectAnswer extends FieldValueDirectAnswer<Email> {
+  fieldType: BuiltInFieldType.Email
+}
+
+/**
+ * A type to use in email direct answer's value {@link FieldValueDirectAnswer.value}.
+ *
+ * @public
+ */
+export type Email = string[];
```

Comment:
I like the idea of making a type in general, but here it feels a little unnecessary? I would probably just do FieldValueDirectAnswer<string[]>

Reply:
also is email always `string[]` or can it also just be `string`?

Reply:
From what I've tested, they all returned `string[]` including results that only have one email.